server/rest/dataset.py

```diff
@@ -400,3 +403,84 @@ def createGeojson(self, item, params):
         .param('endTime', 'latest time to include result', required=False)
         .errorResponse('ID was invalid.')
         .errorResponse('Write permission denied on the Item.', 403))
+
+    @access.public
+    @autoDescribeRoute(
+        Description('Download a dataset.')
+        .modelParam('id', model='item', level=AccessType.READ)
+        .errorResponse('ID was invalid.')
+        .errorResponse('Read access was denied on the parent folder.', 403))
+    def download(self, item, params):
+        minervaMeta = item['meta']['minerva']
```

Comment:
@aashish24, You are right. It needs some code change, but if additional database support is acquired through https://github.com/OpenGeoscience/database_assetstore, only minor change is needed to generalize different database. I'll keep in mind of this in future development.